using-mobile-extensions/adobe-media-analytics/media-api-reference.md

```diff
@@ -55,19 +53,21 @@ ACPMedia.extensionVersion().then(mediaExtensionVersion => console.log("AdobeExpe
 ### createTracker
 
 {% hint style="warning" %}
-The API createTracker with callback has been deprecated for the synchronous version
+The API createTracker with callback has been deprecated.
 {% endhint %}
 
-Creates a media tracker instance that tracks the playback session. The tracker created should be used to track the streaming content and it sends periodic pings to the media analytics backend.
+This API creates a media tracker instance that tracks the playback session. The tracker created should be used to track the streaming content and it sends periodic pings to the media analytics backend.
 
 {% tabs %}
 {% tab title="Android" %}
-#### createTracker
+#### Java
 
-The createTracker function returns the instance of MediaTracker for tracking a media session. The createTracker function with callback as a parameter has been deprecated.
+The createTracker function returns the instance of `MediaTracker` for tracking a media session.
 
 {% hint style="warning" %}
-If MobileCore.resetIdentities\(\) is called in the implementation, the existing tracker will stop sending pings. You will need to create a new tracker to generate a new media session.
+If MobileCore.resetIdentities() is called in the implementation, the existing tracker will stop sending pings. You will need to create a new tracker to generate a new media session.
+
+Please note that the createTracker function with a callback has been **deprecated**.
```

Comment:
THis is a different warning on it's own. How are we handling multiple warnings with different context. Do we club them or is it okay to have them separate?